### ISSUE TITLE
Flynn is confused by how many jobs is running

### ISSUE THREAD

**Issue by kwando** [flynn/flynn#1434]
```
kwando at alice in ~/simple-app on master
$ flynn ps
ID                                      TYPE
flynn-0db072aa69dc4b19b1674fb591bdcfb9  web
flynn-c7f866569b83426697661971890b056f  web

kwando at alice in ~/simple-app on master
$ flynn scale web=1
requested scale equals current scale, nothing to do!
```

https://gist.github.com/1b3181a28f37ec647e14

**Comment by titanous** [comment]
What version of flynn are you running (`flynn-host version` on the server).

**Comment by kwando** [comment]
v20150407.0

**Comment by titanous** [comment]
This is some variant of #686, did it happen after a restart?

**Comment by kwando** [comment]
I think I did ```stop flynn-host``` and then ```start flynn-host```. This procedure almost always brake flynn, a full reboot of the box usually fixes the problems though.

**Comment by temujin9** [comment]
Having similar issues. No restart involved, but it did happen after a rebuild of the application.

**Comment by jzila** [comment]
Might be related to #1182

**Comment by lmars** [comment]
We have made some changes to the CLI (#1505) which should make it easier to diagnose the issue here:

* `flynn ps` will now show the release a process is from
* `flynn release show` will show information on the current release

So using these two commands, you should be able to identify which process should not be running (as it will not be from the current release), and scale that release down (e.g. `flynn scale --release RELEASE_ID web=0`).

**Comment by titanous** [comment]
This should be resolved by the new scheduler (#1914).
